packages/xo-server/src/xo-mixins/logs/loggers/leveldb.js

```diff
@@ -38,7 +38,11 @@ export default class LevelDbLogger extends AbstractLogger {
 
     const key = generateUniqueKey(time)
 
-    const promise = this._db.put(key, log)
+    const promise = new Promise(resolve => {
+      setTimeout(() => {
+        resolve(this._db.put(key, log))
+      }, 3)
+    })
```

Comment:
What's the point of this change?

Reply:
This does not prove our fix works, it only proves that if you're reading the log before writing it, it will not be up to date....

Reply:
The bug is not reproducible. It occurs rarely.

Reply:
The t'es would be meaningful if you keep this change but revert the other one.

Reply:
Does it work now?
Without await it should be interrupted, with it should not.

Reply:
Yes, it works. And the 3 ms is the threshold.